src/rpc/rpc_server.h

```diff
@@ -24,9 +27,14 @@ class RPCServer {
 
     ~RPCServer();
 
+    void OnLvsStored(std::shared_ptr<Milestone> pms);
+
 private:
-    BasicRPCServer server;
-    std::vector<std::unique_ptr<grpc::Service>> service_impls;
+    BasicRPCServer server_;
+    std::vector<std::unique_ptr<grpc::Service>> service_impls_;
+
+    std::function<void(std::shared_ptr<Milestone>)> callback_ = nullptr;
```

Comment:
Where is it assigned?

Reply:
[rpc_server.cpp](https://github.com/EPI-ONE/epic/blob/8e4240b1e84fe83dbadf6d03f44467eec962e876/src/rpc/rpc_server.cpp#L16)